docs/examples/general/reductions.ipynb

```diff
@@ -2,28 +2,28 @@
  "cells": [
```

Comment:
<p>Point 4 misaligned</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I don't see it in the source. Maybe some rendering issue? I'll make sure that it is ok in the final build when I have it.</p>